applications/nrf_desktop/README.rst

```diff
@@ -1682,11 +1682,12 @@ The following example shows the information passed to the plugin metadata:
    GType = FuNordicHidCfgChannel
    NordicHidBootloader = B0
 
-* ``[HIDRAW\VEN_1915&DEV_52DE]``: describes the method of communication (``HIDRAW`` selects communication using ``HID`` reports), vendor, and device identifiers.
-  This information tells the fwupd how to read the version number devices with this identifier and identifies the device.
-* ``Plugin = nordic_hid`` and ``GType = FuNordicHidCfgChannel``: sets the plugin that the device uses.
-* ``NordicHidBootloader``: selects the bootloader that the device is running.
-  ``NordicHidBootloader`` is an optional entry.
+In this console snippet:
+
+* ``[HIDRAW\VEN_1915&DEV_52DE]`` - This line describes the method of communication (``HIDRAW`` selects communication using ``HID`` reports), vendor, and device identifiers.
```

Comment:
I think we need to rephrase it. If we have open window for documentation update let's do it now.

` - This line describes OS provided device instance id that identifies the device`